content/en/news/releases/1.7.x/announcing-1.7.1/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Announcing Istio 1.7.1
+linktitle: 1.7.1
+subtitle: Patch Release
+description: Istio 1.7.1 patch release.
+publishdate: 2020-09-10
+release: 1.7.1
+aliases:
+    - /news/announcing-1.7.1
+---
+
+This release contains bug fixes to improve robustness. This release note describes what’s different between Istio 1.7.0 and Istio 1.7.1
+
+{{< relnote >}}
+
+# Changes
+
+
+
+- **Added** Envoy [ext authz and gRPC access log API support](https://github.com/istio/istio/wiki/Enabling-Envoy-Authorization-Service-and-gRPC-Access-Log-Service-With-Mixer) in Mixer,
```

Comment:
This seems odd to me as it's pointing to a wiki page and doesn't involve any code changes i.e. is there anything specific in 1.7.1 that unlocks this feature?

Reply:
@bianpengyuan

Reply:
Performance testing, upgrade testing and small bug fixes unblocks this feature in 1.7.1.

Reply:
So I guess It should say "Fixed" instead of "Added"?

Reply:
Not really, we did not publish this in 1.7.0 release note, because I think performance and upgrade are requirements for a feature to release and they were not done in time.

Reply:
I see, thanks for the clarification.